<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #EEEDEB;  
}
QLineEdit {
	background-color: #F2EFE5;
    border: 1px solid #cccccc; /* Light Gray Border */
    border-radius: 3px; /* Rounded Corners */
}
QDateEdit {
	background-color: #F2EFE5;
    border: 1px solid #cccccc; /* Light Gray Border */
    border-radius: 3px; /* Rounded Corners */
}
QLineEdit:focus {
    border-color: #007acc; /* Blue border color when focused */
}
QLabel {
    color: rgb(21, 76, 121);
    text-align: center;
}</string>
  </property>
  <widget class="QLabel" name="explain_error">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>39</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>فایل انتخابی باید اکسل باشد !</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #219C90;
color: rgb(34, 40, 49);
border-radius: 4px;



</string>
   </property>
   <property name="text">
    <string>متوجه شدم</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>26</y>
     <width>49</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>icon/11586089.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
